src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -502,6 +503,53 @@ public void getProjectsConciseFilterByTagTest() {
                 """);
     }
 
+    @Test
+    public void getProjectsConciseFilterByTeamTest() {
+        final var projectA = new Project();
+        projectA.setName("acme-app-a");
+        qm.persist(projectA);
+
+        final var projectB = new Project();
+        projectB.setName("acme-app-b");
+        qm.persist(projectB);
+
+        qm.bind(projectB, List.of(qm.createTag("foo")));
+
```

Comment:
Three things missing here:

1. You need to enable the portfolio ACL feature for this test
2. You need to assign `projectB` to the team of the test user
3. To get the assertion in the second request of this test to work, you need to filter for `team.getName()` (`team` is available in every test method already, and it's the team you authenticate as by using `.header(X_API_KEY, apiKey)`).

The former you can achieve like this: https://github.com/DependencyTrack/hyades-apiserver/blob/5de4be3c7ba133758859ffdfd5874ce9e90a7ff5/src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java#L317-L323

I recommend putting this at the very top of your this test method.

The latter you do like this: https://github.com/DependencyTrack/hyades-apiserver/blob/5de4be3c7ba133758859ffdfd5874ce9e90a7ff5/src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java#L333-L334

Reply:
For clarification,  my team and I believe that this can be achieved similar to lines 141-143 of ProjectResourceTest.java?
```suggestion
// Create project and give access to current principal's team.
        final Project accessProject = qm.createProject("acme-app-a", null, "1.0.0", null, null, null, true, false);
        accessProject.setAccessTeams(List.of(team));
```

Reply:
Yeah, those lines are achieving the same. For the tests, it doesn't really matter if you use `Project.setAccessTeams(listOfTeams)` or `Project.addAccessTeam(team)`. Although the former is a bit less verbose IMO.